docs/config/index.md

```diff
@@ -235,6 +235,41 @@ Silent mode
 
 Path to setup files
 
+### globalSetup
+
+- **Type:** `string | string[]`
+
+Path to global setup files, relative to project root
+```js
+{
+    globalSetup: './vitest-global-setup.js'
+}
+```
+
+Files can either export named functions
+```js
+let server;
+export async function setup() {
+    server = await startServer()
+}
+export async function teardown() {
+    await server.close()
+}
+```
+
+or a setup function as default that returns the teardown
+```js
```

Comment:
Could be all in one example (divised by comments to show the different approaches